src/app/config/dapps/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Layer } from '../../../oasis-nexus/api'
+import { Network } from '../../../types/network'
+import { wRose } from './wrose'
+import { TFunction } from 'i18next'
+
+export type DappReference = {
+  description: string
+  label: string
+  url: string
+}
+
+export type DappReferenceGenerator = (t: TFunction) => DappReference
+
+const dappsForToken: Record<Network, Partial<Record<Layer, Record<string, DappReferenceGenerator>>>> = {
+  [Network.testnet]: {
+    [Layer.sapphire]: {
+      [wRose.address]: wRose.app,
```

Comment:
Different address on testnet
https://github.com/oasisprotocol/dapp-wrose/blob/157d7c3/src/constants/config.ts#L14

I'd also add this link in comment

Reply:
Added testnet address and link to the dApp config file